### DISCUSSION TITLE
Fluid properties channel-flow

### DISCUSSION THREAD

**Discussion by rugric** [xcompact3d/Incompact3d#177] [Q&A] [answered]
Can anyone tell me where the fluid properties (density and dynamic viscosity of the fluid) are set in the case of the channel-flow?

**Comment by pbartholomew08** [comment, marked as answer]
This is controlled by the Reynolds number. Xcompact3d uses normalised values / equations so for a given geometry, fluid and reference velocity this will set your Reynolds number. Unless solving a variable density case (e.g. the low Mach number solver) the normalised density will be 1 everywhere.

**Reply by jushi-123** [reply]
Hi, I have some doubt in setting the parameter of turbulent channel flow,especially the steps of rotation and the rotation speed. Is there any tips in setting these two parameters?
